docs/reference/setup/install/targz-security.asciidoc

```diff
@@ -0,0 +1,68 @@
+[role="exclude"]
+==== Start {es} with security enabled
+
+When starting {es} for the first time, security features are enabled and
+configured by default. The following security configuration occurs
+automatically: 
+
+* Authentication and authorization are enabled, and a password is generated for
+the `elastic` built-in superuser.
+* Certificates and keys for TLS are generated for the transport and HTTP layer,
+and TLS is enabled and configured with these keys and certificates.
+
+The password and certificate and keys are output to your terminal. For example:
```

Comment:
Certificates and keys are not printed to the terminal , only the password for elastic user and the enrolment token of Kibana